experiments/compositions/samples/FirstCELComposition/composition/teampage.yaml

```diff
@@ -0,0 +1,165 @@
+# Copyright 2024 Google LLC
```

Comment:
will this scenario be included in the private preview user guide?

Reply:
Not for private preview. I would consider this OSS integration work. And would be part of Public preview.,